Gems/EditorPythonBindings/Code/Source/ActionManager/ActionManagerBus.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/Outcome/Outcome.h>
+#include <AzCore/std/string/string.h>
+
+#include <Source/ActionManager/PythonEditorAction.h>
+
+using ActionManagerOperationResult = AZ::Outcome<void, AZStd::string>;
+
+class ActionManagerRequests : public AZ::EBusTraits
```

Comment:
fix: It looks like you're missing a namespace, all these types are declared in the global scope